x-pack/plugins/lens/common/constants.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const PLUGIN_ID = 'visualization_editor';
```

Comment:
Shouldn't the plugin id reflect the name `lens` in some degree? specially if the plugin folder name is `lens`

Reply:
Will do